data/two_hex_apart.mesh

```diff
@@ -0,0 +1,55 @@
+MFEM mesh v1.0
+
+#
+# MFEM Geometry Types (see mesh/geom.hpp):
+#
```

Comment:
Maybe add a comment about cube dimensions and how far apart they are?

Reply:
Added